### ISSUE TITLE
Dataframe client silently drops additional results

### ISSUE THREAD

**Issue by nkugland** [influxdb/influxdb-python#122]
Just got bitten by unexpected behavior with the dataframe client: 

```
        if len(result) > 0:
            return self._to_dataframe(result[0], time_precision)
        else:
            return result
```

The impact is that results across multiple time series, in which `len(result)>0`, are discarded and only the first one is returned as a dataframe. 

Would it make sense to return a list of data frames in this case? Actually, perhaps it would be better to return one stacked dataframe, in which the timeseries name is a column.

Thanks!

**Comment by ReAzem** [comment]
@timtroendle Would you lke to take a look at this?

**Comment by timtroendle** [comment]
@nkugland The case of multiple time series has indeed not been covered. A list would make sense I guess. By stacked data frame you are talking of single data frame -- kind of a merge of all? As the different time series could have very different time indices, that could lead to many ``NaN``s couldn't it?

Could you provide an example query and JSON result? That would make a test case already.

**Comment by nkugland** [comment]
Yes, by stacked data frame I mean a merge of all, with the time series name as an additional column. My current workaround is to parse each data frame individually, adding the time series name column, and then do a `pandas.concat(...)` on the list of data frames. In my use case I'm only gathering reduced data, so there is no issue with the time indices. 

The quick approach would be to return a list of data frames and let the user handle stacking and merging as they wish. 

Here's a typical query:
```
select mean(value), min(value), max(value), stddev(value)
from /^a94a8fe5ccb19ba61c4c0873d391e987982fbbd3\.*/
where ...
```

...and result set:
```
[	{	u'points': [[0, 323048, 323048, 323048, 0]], 
		u'name': u'a94a8fe5ccb19ba61c4c0873d391e987982fbbd3.series1', 
		u'columns': [u'time', u'mean', u'min', u'max', u'stddev']}, 

	{	u'points': [[0, -2.8233, -2.85034, -2.7832, 0.01732]], 
		u'name': u'a94a8fe5ccb19ba61c4c0873d391e987982fbbd3.series2', 
		u'columns': [u'time', u'mean', u'min', u'max', u'stddev']}, 

		...

	{	u'points': [[0, -0.012207, -0.012207, -0.012207, 0]], 
		u'name': u'a94a8fe5ccb19ba61c4c0873d391e987982fbbd3.seriesN', 
		u'columns': [u'time', u'mean', u'min', u'max', u'stddev']} 
]
```

**Comment by ReAzem** [comment]
This was closed in #123
